vertexai/src/LiveContentResponse.cs

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Google.MiniJSON;
+using Firebase.VertexAI.Internal;
+using System.Linq;
+using System;
+
+namespace Firebase.VertexAI {
+
+/// <summary>
+/// Represents the response from the model for live content updates.
+/// </summary>
+public readonly struct LiveContentResponse {
+  /// <summary>
+  /// Represents the status of the `LiveContentResponse`.
+  /// </summary>
+  public enum LiveResponseStatus {
```

Comment:
since we agree to follow proto, do you want to make the update now or let's do that in a follow up?

Reply:
Let's do it as a follow up, since we need to come to an agreement across all the platforms.